_docs/layout/page-layouts.md

```diff
@@ -23,7 +23,7 @@ resource: true
 - Use the main navigation to navigate between the main sections of the
   application. Make sure your application consistently
   uses either a top navigation or a side navigation, but never both. You can
-  use dropdown for subcategories, but be sure to only use one sub-level.
+  use dropdowns for subcategories, but be sure to only use one sub-level.
```

Comment:
Actually, we should also hyphenate drop-down. We're trying to catch them when we find them.